server/src/main/java/com/google/finapp/FinAppService.java

```diff
@@ -197,12 +197,16 @@ public void getRecentTransactionsForAccount(
     } catch (SpannerDaoException e) {
       responseObserver.onError(Status.fromThrowable(e).asException());
       return;
+    } catch (IllegalArgumentException e) {
```

Comment:
nit: Maybe we can use a multi-catch here since the action we perform is the same for both ?